javascripts/content_scripts/privly.js

```diff
@@ -378,10 +372,11 @@ var privly = {
        };
 
     //Styling and display attributes
-     for(var key in attrs) 
-     {
-       iFrame.setAttribute(key, attrs[key]);
-     }
+	for(var key in attrs) {
```

Comment:
The whitespace shifted here. You also don't need to check the existence of the iframe since it is created in this function.